sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+class AccessPolicy(Model, object):
```

Comment:
@rakshith91, should this inherit Model or contain a model?

Reply:
`AccessPolicy` just exposes the generated AccessPolicy model which inherits from `Model`.
I think its fine.